src/libs/libjpeg_turbo/lv_libjpeg_turbo.c

```diff
@@ -37,16 +38,27 @@ static lv_result_t decoder_open(lv_image_decoder_t * decoder, lv_image_decoder_d
                                 const lv_image_decoder_args_t * args);
 static void decoder_close(lv_image_decoder_t * decoder, lv_image_decoder_dsc_t * dsc);
 static lv_draw_buf_t * decode_jpeg_file(const char * filename);
-static bool get_jpeg_size(const char * filename, uint32_t * width, uint32_t * height);
+static uint8_t * alloc_file(const char * filename, uint32_t * size);
```

Comment:
Because you are already editing this file, could you please rename `alloc_file()` to `read_file()` ?
This name describes better what the function does: reads the file content into memory.

Reply:
thanks for the feedback, I'll update them later.